src/modules/twap/utils/parseTwapError.ts

```diff
@@ -1,26 +1,17 @@
-enum TwapErrorCodes {
-  INVALID_ARGUMENT = 'INVALID_ARGUMENT',
-}
-
-const TwapErrorMessages = {
-  DEFAULT: 'Something went wrong creating your order',
-}
+const DEFAULT_ERROR_MESSAGE = 'Something went wrong creating your order'
 
-function parseInvalidArgumentError(error: any) {
-  const regex = /(?<=\(argument=")[^"]+(?=",)/g
-  const matches = error.message.match(regex)
-  const invalidArgument = matches?.length ? matches[1] : ''
-  const str = invalidArgument ? `"${invalidArgument}" ` : ''
+function getIvalidArgumentError(error: any): string | undefined {
+  if (error && error.message && error.message.includes('INVALID_ARGUMENT')) {
+    const matches = error.message.match(/argument="([^"]+)"/)
+    const invalidArgument = matches?.length ? matches[1] : ''
```

Comment:
Nitpick. Would this work?

```suggestion
    const invalidArgument = matches?.[1] || ''
```